permission_handler/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 11.1.0
```

Comment:
Typically we have CHANGELOG entries for all different packages (`permission_handler`, `permission_handler_apple`, `permission_handler_platform_interface`).